admin-dev/themes/new-theme/js/pages/order/view/order-product-renderer.js

```diff
@@ -46,7 +46,17 @@ export default class OrderProductRenderer {
     $(OrderViewPageMap.productsCount).html(numProducts);
   }
 
-  editProductFromList(orderDetailId, quantity, priceTaxIncl, priceTaxExcl, taxRate, location, availableQuantity, orderInvoiceId) {
+  editProductFromList(
+    orderDetailId,
+    quantity,
+    priceTaxIncl,
+    priceTaxExcl,
+    taxRate,
+    location,
+    availableQuantity,
+    availableOutOfStock,
+    orderInvoiceId
```

Comment:
```suggestion
    orderInvoiceId,
```

Reply:
Euuuuh 😮 in a function call ? A trailing comma for last argument ? Even if it's syntaxically valid, is it something we want ?

Reply:
It works in both cases and in both cases js linter is unhappy, so.. Its your call :smile:

Reply:
I will consider @PierreRambaud knowledge of JS code style better than me 👍

Reply:
eslint-airbnb recommandation :)